src/app/tests/suites/certification/Test_TC_CNET_4_2.yaml

```diff
@@ -193,3 +193,45 @@ tests:
           value: null
           constraints:
               type: int32s
+
+    # Issue: https://github.com/project-chip/connectedhomeip/issues/30081
+    - label:
+          "Step 14: TH reads the SupportedThreadFeatures attribute from the DUT"
```

Comment:
as above

Reply:
Getting Unsupported attribute response Issue need to be fixed https://github.com/project-chip/connectedhomeip/issues/30081 to make this step automated.